backend-lib/src/main/java/cash/z/ecc/android/sdk/internal/jni/RustBackend.kt

```diff
@@ -617,6 +631,14 @@ class RustBackend private constructor(
             networkId: Int
         ): String
 
+        @JvmStatic
+        private external fun getNextAvailableAddress(
+            dbDataPath: String,
+            accountUuid: ByteArray,
+            networkId: Int,
+            receiverFlags: Int
```

Comment:
In the majority of existing JNIs, `networkId` is the last argument.
```suggestion
            receiverFlags: Int,
            networkId: Int
```